content/zh/blog/_posts/2022-02-17-updated-dockershim-faq.md

```diff
@@ -345,15 +366,15 @@ Kubernetes documentation on [Container Runtimes].
 <!--
 ### What if I have more questions?
 -->
-### 我还有其他问题怎么办？
+### 我还有其他问题怎么办？ {#what-if-i-have-more-questions}
 
 <!--
 If you use a vendor-supported Kubernetes distribution, you can ask them about
 upgrade plans for their products. For end-user questions, please post them
-to our end user community forum: https://discuss.kubernetes.io/. 
+to our end user community forum: https://discuss.kubernetes.io/.
 -->
 如果你使用了供应商支持的 Kubernetes 发行版，你可以咨询供应商他们产品的升级计划。
-对于最终用户的问题，请把问题发到我们的最终用户社区的论坛：https://discuss.kubernetes.io/。
+对于最终用户的问题，请把问题发到我们的最终用户社区的论坛：<https://discuss.kubernetes.io/>。
```

Comment:
```suggestion
对于最终用户的问题，请把问题发到我们的最终用户社区的论坛：https://discuss.kubernetes.io/。
```

Reply:
It will not be recognized as URL if `<link>` syntax is missed.

Please refer to:
- <https://kubernetes.io/blog/2022/02/17/dockershim-faq/#what-if-i-have-more-questions>
- <https://kubernetes.io/zh/blog/2022/02/17/dockershim-faq/#我还有其他问题怎么办> (no link, fixed)
- <https://kubernetes.io/blog/2020/12/02/dockershim-faq/#what-if-i-have-more-questions>
- <https://kubernetes.io/zh/blog/2020/12/02/dockershim-faq/#what-if-I-have-more-question> (no link, I will fix-up later)

Reply:
I'm not sure about this because I don't know if markdown will treat it as a custom HTML tag and render it into empty string. It is okay to reformat this explicitly. I.e.

```
对于最终用户的问题，请把问题发到我们的最终用户社区的[论坛](https://discuss.kubernetes.io/)。
```

Reply:
I'm sure this is standard Markdown syntax for URLs.

Ref 1: <https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses>
Ref 2: <https://daringfireball.net/projects/markdown/syntax#autolink>

Reply:
But in this repo only [README](https://github.com/kubernetes/website/blob/main/README.md?plain=1#L48) and [one 2018 blog](https://github.com/kubernetes/website/blob/main/content/en/blog/_posts/2018-07-10-coredns-ga.md?plain=1#L170-L180) use this syntax.

CC @sftim, do you think it's acceptable to use this syntax here?

Reply:
@Sean-n It is good to know that this syntax works. See [preview](https://deploy-preview-33783--kubernetes-io-main-staging.netlify.app/zh/blog/2020/12/02/dockershim-faq/). However, we must be very cautious when using angle brackets `<` or `>`. Sometimes it is not a matter of what the standard says, but what the engine generates. In other words, an engine may have bugs.

Reply:
Okay, I've changed to the common syntax.